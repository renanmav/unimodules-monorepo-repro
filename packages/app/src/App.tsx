import React from 'react';
import {SafeAreaView, StatusBar} from 'react-native';

import {Text} from '@MyApp/ui/common';

const App: React.FC = () => {
  return (
    <>
      <StatusBar barStyle="dark-content" />
      <SafeAreaView testID="safe-area-view">
        <Text>It's working</Text>
      </SafeAreaView>
    </>
  );
};
export default App;
