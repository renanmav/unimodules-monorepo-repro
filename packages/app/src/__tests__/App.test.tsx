import React from 'react';
import {render} from '@testing-library/react-native';
import App from '../App';

it('should render correctly', async () => {
  const {toJSON} = render(<App />);

  expect(toJSON()).toMatchSnapshot();
});
