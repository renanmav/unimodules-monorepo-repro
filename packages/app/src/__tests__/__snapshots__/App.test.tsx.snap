// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  testID="safe-area-view"
>
  <Text>
    It's working
  </Text>
</RCTSafeAreaView>
`;
